### PR TITLE
Add instructions for adding custom ssl cert to trust store

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?
Add instructions for adding custom ssl cert to trust store

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16002

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+. Set the `global.customPublicCert` property to `true`.
++
+=====
+.If {prod-short} is deployed using a link:https://helm.sh/[Helm Chart]
+. Clone the https://github.com/eclipse/che[che] project
+. Go to `deploy/kubernetes/helm/che` directory
+. Update the `global.tls.useCustomPublicCerts` property. To do that, add the following option to the `helm upgrade` command:
++
+----
+$ helm upgrade che --set global.tls.useCustomPublicCerts=true --set global.ingressDomain=$(minikube ip).nip.io .
```
Comment (tolusha) [inline src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: Is it only about minikube?
It is better
```
-set global.ingressDomain=<kubernetes_cluster_domain>
```
Have a look at other pages where `global.ingressDomain` is mentioned.

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.Procedure
+. Save the certificate that you want to apply to `ca.crt` file.
+. Create a new *configMap* with the certificate:
++
+[subs="+quotes"]
+----
+$ kubectl create configmap custom-public-cert --from-file=ca.crt -n=che
+----
+. Set the `global.customPublicCert` property to `true`.
++
```
Comment (tolusha) [inline src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc:16]: There is nothing about this property in lines below.
How we set it to `true` ?

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+=====
+.If {prod-short} is deployed using a link:https://helm.sh/[Helm Chart]
+. Clone the https://github.com/eclipse/che[che] project
+. Go to `deploy/kubernetes/helm/che` directory
+. Update the `global.tls.useCustomPublicCerts` property. To do that, add the following option to the `helm upgrade` command:
```
Comment (tolusha) [inline src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: IMHO:
Set the `global.tls.useCustomPublicCerts`  property to `true` in  {prod-short} deployment with the following command:

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
++
+----
+$ helm upgrade che --set global.tls.useCustomPublicCerts=true --set global.ingressDomain=$(minikube ip).nip.io .
+----
+=====
++
+====
+.If {prod-short} is deployed using link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operators]
+Update the `gitSelfSignedCert` property. To do that, execute:
+
```
Comment (tolusha) [inline src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc:33]: `customPublicCert` ?

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
+[id="adding-custom-certificates-to-trust-store_{context}"]
+= Adding custom public SSL certificates to {prod-short} trust-store
+
+This procedure describes how to configure {prod-short} to be able to handle HTTP requests from untrusted sources.
+
+.Procedure
+. Save the certificate that you want to apply to `ca.crt` file.
+. Create a new *configMap* with the certificate:
```
Comment (tolusha) [inline src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: configMap or secret?

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 1 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: Could you please refer to that place?

Comment (tolusha) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: https://github.com/eclipse/che-docs/blob/che-16002/src/main/pages/che-7/administration-guide/proc_deploying-registries-in-kubernetes.adoc#L20

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 2 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: The instruction is below

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 5 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: ConfigMap, I don't see any reasons to use secret

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 3 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: fixed

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 4 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc:33]: fixed

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 2 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc:16]: fixed the title

File: src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc (hunk 1 above)
Comment (vinokurig) [reply src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc]: fixed

**Comment by vinokurig** [comment]
@tolusha @themr0c @rkratky
I've fixed all the notes, could you please review

**Review by tolusha — APPROVED** [review]
It is ok for me.
Should be merged once code is merged.

**Comment by themr0c** [comment]
What do you mean by "once code is merged"?

**Comment by vinokurig** [comment]
I guess it means the related PR is merged: https://github.com/eclipse/che-operator/pull/171 and https://github.com/eclipse/che/pull/16006

**Pull request merged**